includes/adapter/shipment-options-from-order-adapter.php

```diff
@@ -132,15 +108,10 @@ private function isReturnShipmentFromOptions(array $options, ?AbstractShipmentOp
      */
     private function isAgeCheckFromOptions(array $options, ?AbstractShipmentOptionsAdapter $shipmentOptionsAdapter): ?bool
     {
-        if (array_key_exists('age_check', $options)) {
-            return (bool) $options['age_check'];
-        }
+        $valueFromOptions = (bool) ($options['age_check'] ?? null);
```

Comment:
die (bool) hier gaat niet werken, nu is $valueFromOptions altijd true/false en kan hij nooit terugvallen naar $valueFromAdapter, je kunt wel dit doen:

```
return (bool) ($valueFromOptions ?? $valueFromAdapter);
```